src/qubex/experiment/mixin/characterization_mixin.py

```
@@ -1397,16 +1415,132 @@ def measure_phase_shift(
         phase_shift = coefficients[0]
         return phase_shift
 
+    def measure_electrical_delay(
```

Pylint (3.3.7)

[warning] Number of parameters was 8 in 'CharacterizationProtocol.measure_electrical_delay' and is now 10 in overriding 'CharacterizationMixin.measure_electrical_delay' method

(W0221)